specs/Checkbox.md

```diff
@@ -0,0 +1,388 @@
+| inputProps           | React.ButtonHTMLAttributes<HTMLElement or HTMLButtonElement>|                                                                                |
+| keytipProps          | IKpeytipProps                                               |                                                                                |  
+| label                | string                                                      |                                                                                |
+| onChange             | (ev, checked) => void                                       |                                                                                |
+| onRenderLabel        | IRenderFunction<ICheckboxProps>                             |                                                                                |
+| styles               | IStyleFunctionOrObject<ICheckboxStyleProps, ICheckboxStyles>|                                                                                |
+| theme                | ITheme                                                      |                                                                                |
+
+### Stardust Checkbox props
+
+
+| Name                 | Type                                                        | Notes                                                                          |
+| -------------------- | --------------------------------------------------------    | -------------------------------------------------------------------------------|
+| animation            | AnimationProp                                               |                                                                                |
+| as                   | React.ElementType                                           | default type is "div"                                                          |
+| className            | string                                                      |                                                                                |
+| content              | ReactNode                                                   |                                                                                |
+| design               | ComponentDesign                                             |                                                                                |
+| disableAnimations    | boolean                                                     | default false                                                                  |
+| overwrite            | boolean                                                     | default false                                                                  |
+| renderer             | Renderer                                                    |                                                                                |
```

Comment:
Don't think this is a prop.  Double check?

Reply:
Yup removed.